graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -90,11 +94,14 @@ def __init__(
         self.filesystem_type = filesystem_type
         if self.filesystem_type == "s3":
             self.bucket = self.input_prefix.split("/")[1]
-            self.pyarrow_fs = fs.S3FileSystem(region=region)
+            self.pyarrow_fs = fs.S3FileSystem(
+                region=region, retry_strategy=fs.AwsDefaultS3RetryStrategy(max_attempts=10)
+            )
         else:
             self.pyarrow_fs = fs.LocalFileSystem()
         self.verify_outputs = verify_outputs
         self.streaming_repartitioning = streaming_repartitioning
+        self.verbosity = 1 if logging.getLogger().getEffectiveLevel() <= logging.INFO else 0
```

Comment:
Why not use logging to control?

Reply:
joblib has its own verbosity level, which is an int, 0 being no logging. We impute how much logging the user wants from the log level they have chosen. I actually think I'll set it to 0 and only increase to >0 for logging.DEBUG level.